tests/support/pretest.js

```diff
@@ -0,0 +1,127 @@
+/*jshint node: true */
+var _ = require('lodash'),
+	/*jshint -W079 */
+	Promise = require('bluebird'),
+	atob = require('atob'),
+	/*jshint +W079 */
+	fs = Promise.promisifyAll(require('fs-extra')),
+	glob = Promise.promisify(require('glob')),
+	path = require('path'),
+	pkg = require('../../package.json')['web-platform-tests'],
+	request = Promise.promisifyAll(require('request')),
+	url = require('url');
+
+var testPath = path.resolve(path.join(__dirname, '..', '..', pkg.path)),
+	token = 'Mjk5ZGQxNTk0ZDA3YTllY2I5YzlmMzRhZWYyOTEyZTQ1MDE2ZDdmNw==',
```

Comment:
It's a no-scope public token (so harmless, readonly access). It allows getting the higher 50k per hour request limit.